src/handler.py

```diff
@@ -42,19 +37,19 @@ def connection_manager(event, context):
         # Add connectionID to the database
         table = dynamodb.Table(jobsConnectionTable)
         table.put_item(Item={"ConnectionID": connectionID})
-        return _get_response(200, "Connect successful.")
+        return get_response(200, "Connect successful.")
```

Comment:
A while ago I came across the python `http` module, which gives some nice enumerations for HTTP statuses. Rather than using 200, you can use HTTPStatus.OK and it should still have the same behavior.

Obviously HTTP status codes aren't going to change overnight, but it's great to rely on an enumeration like this for readability and standardization.

https://docs.python.org/3/library/http.html#http.HTTPStatus